app.rb

```diff
@@ -1,8 +1,19 @@
 require 'sinatra/base'
+require './lib/peep'
+require './database_connection_setup'
 
 class Chitter < Sinatra::Base
-  get '/test' do
-    'Test page'
+
+  get '/messages' do
+    Peep.toggle_order if params['sorting']
+    @all_peeps = Peep.all(params['filter_keyword'])
+    erb :messages
+  end
+
+  post '/messages/peep' do
+    new_peep = params[:new_peep]
+    Peep.create(new_peep) unless new_peep.empty?
+    redirect '/messages'
```

Comment:
The restful routes for this app would be...

`get "/peeps"` and `post "/peeps"` to get all peeps and create a peep, respectively.